common/static/common/js/spec_helpers/discussion_spec_helper.js

```diff
@@ -53,7 +53,7 @@
         DiscussionSpecHelper.makeCourseSettings = function() {
             return new DiscussionCourseSettings({
                 category_map: {
-                    children: ['Test Topic', 'Other Topic'],
+                    children: [['Test Topic', 'entry'], ['Other Topic', 'entry']],
```

Comment:
@awaisdar001: I think the problem here is that tuples make more sense in Python, but get translated to arrays in JavaScript.  Switching to a dict would make more sense in both Python and JavaScript, but that would be a larger change.  Switching to an array in Python would make this more consistent, but would make less sense in Python.  So, if I were going to hold thumbs for consistency, I'd rather see this move to:
```
children: [
    {type: 'entry', name: 'Test Topic'}, 
    {type: 'entry', name: 'Other Topic'}]
],
```
1. @awaisdar001: I'm ok with this staying as-is, even though I would prefer to see dict/json used as discussed elsewhere.  I'll leave it to you to determine if you want to push for the cleaner code.
2. If we get the cleaner code (which would read much nicer), I think it would be nice to have `type` first, as I did above, but that's a nice to have.